src/components/header/Header.tsx

```diff
@@ -13,25 +13,17 @@ type HeaderLayoutProps = {
 export const HeaderLayout = ({ collapsed, toggleCollapsed }: HeaderLayoutProps) => {
   const user = AuthService.instance.getAuth();
 
-  const {
-    token: { colorBgContainer },
-  } = theme.useToken();
+  const { token } = theme.useToken();
 
   return (
-    <HeaderLayoutStyled style={{ background: colorBgContainer }}>
+    <HeaderLayoutStyled style={{ background: token.colorBgContainer }}>
```

Comment:
@phanvohieunghia  I think not using an inline style.
should move it into a styled file for all. 🤔